global_helpers/panther_base_helpers.py

```diff
@@ -58,3 +58,29 @@ def is_dmz_tags(resource):
 # function being used, etc.
 IN_PCI_SCOPE = in_pci_scope_tags
 IS_DMZ = is_dmz_tags
+
+GSUITE_PARAMETER_VALUES = [
+    'value',
+    'intValue',
+    'boolValue',
+    'multiValue',
+    'multiIntValue',
+    'messageValue',
+    'multiMessageValue',
+]
+
+
+# GSuite parameters are formatted as a list of dictionaries, where each dictionary has a 'name' key
+# that maps to the name of the parameter, and one key from GSUITE_PARAMETER_VALUES that maps to the
+# value of the parameter. This means to lookup the value of a particular parameter, you must
+# traverse the entire list of parameters to find it and then know (or guess) what type of value it
+# contains. This helper function handles that for us.
```

Comment:
Can you maybe add an example of what the input/output should be? It's hard to picture

Reply:
Sure, I added an example. Let me know if that clears things up a bit.

Reply:
Definitely